rspace/src/main/scala/coop/rchain/rspace/pure/PureSpace.scala

```diff
@@ -0,0 +1,28 @@
+package coop.rchain.rspace.pure
+
+import coop.rchain.catscontrib.Capture
+import coop.rchain.rspace._
+
+import scala.collection.immutable.Seq
+
+class PureSpace[F[_], C, P, A, K](space: ISpace[C, P, A, K]) {
```

Comment:
If there's going to be a `ReplayRSpace` shouldn't we try to keep naming consistent and call this one `PureRSpace` ?